src/views/AuthoritiesSearch/Reports/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Reports';
```

Comment:
Please use named export to make sure consistent naming of component where it's imported